content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md

```diff
@@ -1,1440 +1,1362 @@
+
+![Portenta X8 SSH Session with PuTTY - Setup](assets/ssh-x8-putty.png "Portenta X8 SSH Session with PuTTY - Setup")
+
+In the PuTTY Configuration window, keeping the default values, you must specify the _Host Name (or IP address)_ field with `portenta-x8-<UUID>`. For instance, you would use:
+
+```
+portenta-x8-1822aa09dab6fad9
+```
+
+Click on `Open`, and it will prompt a security alert. It displays information about the connection, including fingerprint details. Depending on your connection profile preference, you can choose to `Accept` or `Connect Once`.
+
+![Portenta X8 SSH Session with PuTTY - Authentication](assets/ssh-x8-putty-auth.png "Portenta X8 SSH Session with PuTTY - Authentication")
+
+After verifying the security alert and proceeding, you have an SSH session that has begun communicating with the Portenta X8.
+
+![Portenta X8 SSH Session with PuTTY](assets/ssh-x8-putty-session.png "Portenta X8 SSH Session with PuTTY")
+
+### Inspect Real-Time Tasks And Logs Via CLI
+
+Run `journalctl -f` to check the status of active services and possibly their errors, as well as various system event logs.
+
+![ADB shell journalctl package](assets/adb-shell-real-time-tasks.png "ADB shell journalctl package")
+
+By calling `journalctl`, it is possible to look at the log of all the running activities by specifying the type of log you are looking for. Some logs may be a warning that can be ignored, while some may be critical errors. Type `journalctl -p 0` to view emergency system messages; otherwise, change the number 0 with the error code you want to investigate according to the following error code numbers:
+
+| **Error code** | **Meaning** |
+|----------------|-------------|
+| 0              | Emergency   |
+| 1              | Alerts      |
+| 2              | Critical    |
+| 3              | Errors      |
+| 4              | Warning     |
+| 5              | Notice      |
+| 6              | Info        |
+| 7              | Debug       |
+
+When you specify the error code, it shows all messages from that code and above. For example, if you specify error code 2, it shows all messages with priority 2, 1 and 0.
+
+You can also view logs for a specific time and date duration. You can use the `-- since` switch with a combination of `"yesterday"`, `"now"`, or a specific date and time `"YYYY-MM-DD HH:MM:SS"`.
+
+An example of how to use the command:
+
+```bash
+journalctl --since "2022-12-22 12:20:00" --until yesterday
+```
+
+### Developing and Deploying Custom Docker Containers on Portenta X8
+
+To customize the Portenta X8, you can develop and deploy Docker containers. Start by provisioning your device as described in the [**Portenta X8 with Arduino Cloud**](#portenta-x8-with-arduino-cloud) section.
```

Comment:
I find this really confusing. It seems that you need to connect to Arduino Cloud to deploy a container, which is not the case. Could we not rephrase this section with a different container example avoiding Arduino Cloud? It might be any of them, available on docker hub, but not the Arduino Cloud one

Reply:
Understood, this was a bit tricky but it is now updated!